src/animations.css

```diff
@@ -20,6 +20,7 @@
 .animation-spin { animation: spin 1.5s linear; }
 .animation-fade-in { animation: fadein 1.5s ease-in forwards; }
 .animation-fade-out { animation: fadein 1.5s ease-out reverse forwards; }
+.animation-fade-in-out { animation: fadeinout 1.5s linear forwards; }
```

Comment:
Can you add an example to the JSdoc comment so we get something to show up on the docs site?

Reply:
Jeepers, sure thing